[DOCS-3072] APM Tracing Ingestion Mechanisms documentation (#12905)
DataDog/documentation: content/en/tracing/visualization/_index.md

{{< img src="tracing/visualization/span_with_metadata.png" alt="span" >}}

### Service entry span

A span is a service entry span when it is the entrypoint method for a request to a service. You can visualize this within Datadog APM when the color of the immediate parent on a flame graph is a different color. Services are also listed on the right when viewing a flame graph.

> ### Trace Root span

Reviewer: ```suggestion
### Trace root span
```

> A span is a root span when it is the first span of a trace. The root span is the entrypoint method of the traced request, its start marks the beginning of the trace.

Reviewer: ```suggestion
A span is a trace root span when it is the first span of a trace. The root span is the entry-point method of the traced request. Its start marks the beginning of the trace.
```

For the example below, the **service entry spans** are:

> - rack.request (which is also the **root span**)
> - aspnet_coremvc.request
> - The topmost green span below aspnet_coremvc.request
> - Every orange mongodb span

Reviewer: ```suggestion
- `rack.request` (which is also the _root span_)
- `aspnet_coremvc.request`
- The topmost green span below `aspnet_coremvc.request`
- Every orange `mongodb` span
```
